### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by AndrewKralovec**
For the games page, the initial load takes a couple of seconds. From a users point of view, I thought the page was not working (even navigated away thinking it was down). I believe adding a loader to the page, letting the user now its fetching data, would result in a better experience. 

Page load time:
GamePerformance

Loader preview
image

https://www.fantasycritic.games/games

### CODE REVIEW

**Comment by SteveF92** [comment]
I think this is a good idea, but I think it can be done much simpler if we use font awesome, which is already used across the rest of the site. Let me know if that's something you want to tackle. Otherwise, I'll merge this and make that change myself.

Also, thanks so much! This will be the first pull request the site's ever had.

File: FantasyCritic.Web/ClientApp/components/pages/masterGames.vue [hunk 1]
```diff
@@ -71,6 +71,7 @@
           </template>
         </b-table>
       </div>
+      <Loader :loading="!showGames" />
```
Comment (SteveF92) [inline FantasyCritic.Web/ClientApp/components/pages/masterGames.vue]: Right here is where we could use font-awesome.

<font-awesome-icon icon="circle-notch" size="4x" spin />

Comment (AndrewKralovec) [reply FantasyCritic.Web/ClientApp/components/pages/masterGames.vue]: Or just use use the spinner icon already defined, your call. 

`<font-awesome-icon icon="spinner"  size="4x" pulse  v-if="!showGames" />`

Comment (SteveF92) [reply FantasyCritic.Web/ClientApp/components/pages/masterGames.vue]: That sounds good to me. Update the PR and I will merge it.

**Pull request merged**